<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>611</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brighntessSlider">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="sliderPosition">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionTest"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionBrightness"/>
    <addaction name="actionContrast"/>
    <addaction name="actionSaturation"/>
    <addaction name="actionHue"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShow_preview"/>
    <addaction name="actionResize_preview"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCreate_Preset"/>
    <addaction name="actionDelete_preset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPresets"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save </string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionSaturation">
   <property name="text">
    <string>Saturation</string>
   </property>
  </action>
  <action name="actionHue">
   <property name="text">
    <string>Hue</string>
   </property>
  </action>
  <action name="actionCreate_Preset">
   <property name="text">
    <string>Add preset</string>
   </property>
  </action>
  <action name="actionDelete_preset">
   <property name="text">
    <string>Delete preset</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
  <action name="actionShow_preview">
   <property name="text">
    <string>Show preview</string>
   </property>
  </action>
  <action name="actionResize_preview">
   <property name="text">
    <string>Resize preview</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>brighntessSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
